quick-start/src/main/ui/app/components/flows-new/edit-flow/mapping/ui/mapping-ui.component.html

```diff
@@ -0,0 +1,123 @@
+<div layout-padding layout="column" class="map-page"> <!--layout-align="center center"> -->
+
+  <div class="map-container2">
+
+    <div id="map-heading">
+
+      <!-- Save/Reset happens via header buttons -->
+      <div id="save-map">
+        <mdl-button mdl-button-type="raised" mdl-colored="primary" mdl-ripple ng-class="btn-save-map" [disabled]="!mapChanged()" (click)="mapChanged() && onSaveMap()">Save</mdl-button>
```

Comment:
As soon as we copy the component for mapping, can we please get read of MDL dependencies?